src/components/ColumnList.tsx

```diff
@@ -1,18 +1,54 @@
+import { Box } from '@mui/material'
+import { Droppable } from 'react-beautiful-dnd'
 import { WithID } from '../helpers/types'
 import { Card, Column } from '../store/store'
 import { ColumnItem } from './ColumnItem'
 
 interface Props {
   cards: WithID<Card>[]
   columns: WithID<Column>[]
+  columnsOrder: string[]
 }
 
-export const ColumnList = ({ cards, columns }: Props) => {
+export const ColumnList = ({ cards, columns, columnsOrder }: Props) => {
+  const filterCards = (columnCardsIds: string[], allCards: WithID<Card>[]) => {
+    const filteredCards = columnCardsIds.map((cardId) => {
+      const [value] = allCards.filter((card) => cardId === card.id)
```

Comment:
Maybe use `find` instead of `filter`?

Reply:
With current solution type for value is `WithID<Card>` and when i change `filter `for `find `I get `WithID<Card> | undefined` which makes trouble later. Maybe we can stay with `filter`?